src/styles.scss

```diff
@@ -0,0 +1,16 @@
+/* You can add global styles to this file, and also import other style files */
+@tailwind base;
+@tailwind components;
```

Comment:
No need for tailwind components unless required

Reply:
removed @tailwind components